GIC-Institution/auth/V4__CREATE_AGGREGATE_RULES_AND_USER.sql

```diff
@@ -101,5 +101,5 @@ SET @uuidConn = REPLACE(UUID(),'-','');
 INSERT INTO `connection` VALUES (unhex(@uuidConn), 'Manual Token Connection', 'manual-token','','');
 
 SET @uuidUser = REPLACE(UUID(),'-','');	
-INSERT INTO user VALUES (unhex(@uuidUser), null, null, null, unhex(@uuidConn),'CommonAreaUser',0,concat('PIC_SURE_USER|', REPLACE(UUID(),'-','')),1,null);
```

Comment:
Can I not count or is this adding one to many?

Reply:
ADd column names